modules/phase_field/doc/content/source/materials/GrandPotentialInterface.md

```diff
@@ -0,0 +1,47 @@
+# GrandPotentialInterface
+
+!syntax description /Materials/GrandPotentialInterface
+
+The multiphase Grand Potential model is parameterized using a bulk free energy
+coefficient $\mu$, a gradient interface coefficient $\kappa$, and a set of
+interface pair coefficients $\gamma_{ij}$.
```

Comment:
Since this material will also calculate grain boundary energies (as done in the test you added), I think it would be good in the documentation to index \gamma and \sigma with {\alpha i \beta j}, where \alpha and \beta represent phases and i and j represent grains, as was done in the paper